spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/TraceScope.java

```diff
@@ -0,0 +1,86 @@
+package org.springframework.cloud.sleuth;
+
+import java.io.Closeable;
+
+import lombok.SneakyThrows;
+import lombok.Value;
+import lombok.experimental.NonFinal;
```

Comment:
I'm sorry if I'm out of line and I should only comment on the trace-id related stuff but is it "safe" to use experimental features of Lombok in a library of Spring-Cloud caliber?